Misc/NEWS.d/next/Tests/2020-07-30-14-08-58.bpo-41439.yhteoi.rst

```diff
@@ -0,0 +1 @@
+Add os.fork checking for test_ssl.BasicSocketTests.test_random_fork and test_uuid.TestUUIDWithExtModule.testIssue8621
```

Comment:
I suggest to be more honest and explain that this change port test_ssl and test_uuid to VxWorks.